ExtraFiles/common/decisions/tesb.txt

```diff
@@ -1080,4 +1080,34 @@ aotearoa_decisions = {
 			}
 		}
 	}
+}
+
+mexhico_decisions = {
+	MEX_object_to_UTA_interference = {
+		visible = {
+			has_idea = MEX_strengthen_democracy
+			country_exists = UTA
+			NOT = { has_flag = MEX_objected_to_UTA_interference }

```

Comment:
You can use `fire_only_once = yes` instead of checking a flag if the flag isn't used for anything else.